js/interface.js

```diff
@@ -84,6 +84,10 @@ var DynamicLists = (function() {
 
     _this.attachListeners();
     _this.init();
+
+    Fliplet.Registry.set('datasource-provider', function() {
```

Comment:
This will do.

```suggestion
    Fliplet.Registry.set('datasource-provider', dataSourceProvider);
```

Reply:
I tried setting it without function but it didn't work because before `dataSourceProvider` initialization it is `undefined`.
Otherwise if I set it after initialization I can't get in on init in `interface-list` because it is still not set.
Setting it as callback let us resolve this value when it is set and use `Registry` on inits.
That's why I created event that triggers when DSP is initialized in `interface` and then I can perform needed actions in `interface-list`.

Reply:
@ivandevupp Thanks for explaining. That makes sense now 👍🏻